src/main/java/com/exasol/adapter/installer/Installer.java

```diff
@@ -0,0 +1,156 @@
+package com.exasol.adapter.installer;
+
+import static com.exasol.adapter.installer.VirtualSchemaInstallerConstants.LINE_SEPARATOR;
+
+import java.io.FileNotFoundException;
+import java.nio.file.Path;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.concurrent.TimeoutException;
+import java.util.logging.Logger;
+
+import com.exasol.bucketfs.BucketAccessException;
+import com.exasol.bucketfs.WriteEnabledBucket;
+
+import lombok.Builder;
```

Comment:
I just added it for development time, because we're adding and renaming fields too often. I can add a normal builder and getters when the structures are stable